Metadata/Driver/AnnotationDriver.php

```diff
@@ -175,7 +180,7 @@ public function loadMetadataForClass(\ReflectionClass $class)
                         $params[] = $this->convertReferenceValue($paramName, $annot->params[$paramName]);
                     }
 
-                    if (!$params) {
+                    if (!$params && !$annot instanceof Call) {
                         continue;
```

Comment:
Yeah, the ``@AfterSetup`` annotation requires a Symfony patch (https://github.com/schmittjoh/symfony/commit/1382f91bc7aa5390d9bbbad3e3daa48293241012), but what we could do is just add a method call which has almost identical behavior.

The reason why I think a separate annotation makes sense is that you want that call to happen at the very end (after everything has been injected), so a separate annotation can make sure of that.